src/main.ts

```diff
@@ -1,13 +1,37 @@
-import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { withInterceptorsFromDi, provideHttpClient } from '@angular/common/http';
+import { enableProdMode, importProvidersFrom } from '@angular/core';
+import { BrowserModule, bootstrapApplication } from '@angular/platform-browser';
+import { provideRouter } from '@angular/router';
+import { EffectsModule } from '@ngrx/effects';
+import { StoreModule } from '@ngrx/store';
+import { HttpClientInMemoryWebApiModule } from 'angular-in-memory-web-api';
 
-import { AppModule } from './app/app.module';
+import { AppComponent } from './app/app.component';
+import { appRoutes } from './app/app.routes';
+import { extModules } from './app/build-specifics';
+import { ProductData } from './app/products/product-data';
+import { ProductEffects } from './app/products/state/product.effects';
+import { productReducer } from './app/products/state/product.reducer';
+import { userReducer } from './app/user/state/user.reducer';
 import { environment } from './environments/environment';
 
 if (environment.production) {
     enableProdMode();
 }
 
-platformBrowserDynamic()
-    .bootstrapModule(AppModule)
-    .catch((err) => console.error(err));
+bootstrapApplication(AppComponent, {
+    providers: [
+        provideRouter(appRoutes),
+        provideHttpClient(withInterceptorsFromDi()),
+        importProvidersFrom(
+            BrowserModule,
+            StoreModule.forRoot({}),
```

Comment:
there are NgRx standalone APIs

e.g. provideStore, provideEffects

https://dev.to/ngrx/using-ngrx-packages-with-standalone-angular-features-53d8